source/standard_scores.Rmd

```diff
@@ -0,0 +1,1523 @@
+The units for the variance, in our case, would be *squared dollars*.  But we
+are more interested in the typical *deviation*, in our original units --
+dollars rather than squared dollars.
+
+So we take the square root of the mean squared deviation (the square root of
+the variance), to get the *standard deviation*.  It is the *standard* deviation
+in the sense that it a measure of *typical* deviation, in the specific sense of
+the square root of the mean squared deviations.
+
+```{python}
+# The standard deviation is the square root of the mean squared deviation.
+# (and therefore, the square root of the variance).
+standard_deviation = np.sqrt(mean_squared_deviation)
+standard_deviation
+```
+
+```{r}
+# The standard deviation is the square root of the mean squared deviation.
+# (and therefore, the square root of the variance).
+standard_deviation <- sqrt(mean_squared_deviation)
+standard_deviation
+```
+
+The *standard deviation* (the square root of the mean squared deviation) is
+a popular alternative to the Mean Absolute Deviation, as a measure of typical
+spread.
+
+@fig-mean-stds shows another histogram of the `income` values, marking the
+mean, the mean plus or minus one standard deviation, and the mean plus or minus
+two standard deviations.  You can see that the mean plus or minus one standard
+deviation includes a fairly large proportion of the data.  The mean plus or
+minus two standard deviation includes much larger proportion.
+
+```{python, eval=TRUE, echo=FALSE, label=fig-mean-stds, fig.cap='Income histogram plus or minus 1 and 2 standard deviations'}
+m = mean_income
+s = get_var('standard_deviation')
+plt.hist(incomes, bins=25, color='gray')
+plt.axvline(m, color='green', linestyle='-', label='Mean')
+plt.axvline(m - 2 * s , color='red', linestyle=':', label='Mean minus 2 SDs')
+plt.axvline(m - 1 * s , color='orange', linestyle=':', label='Mean minus 1 SD')
+plt.axvline(m + 1 * s , color='blue', linestyle=':', label='Mean plus 1 SD')
+plt.axvline(m + 2 * s , color='black', linestyle=':', label='Mean plus 2 SDs')
+plt.legend()
+```
+
+We can use the standard deviations to return to the question of how unusual our
+two congressional districts of interest are in terms of the distribution, by
+calculating how many standard deviations each district is from the mean:
+
+```{python}
+print('Deviation in standard deviation units for KOM:',
+      deviations[kom_index] / standard_deviation)
+print('Deviation in standard deviation units for AOC:',
+      deviations[aoc_index] / standard_deviation)
+```
+
+```{r}
+message('Deviation in standard deviation units for KOM: ',
+        deviations[kom_index] / standard_deviation)
+message('Deviation in standard deviation units for AOC: ',
+        deviations[aoc_index] / standard_deviation)
+```
+
+The values for each district are a re-expression of the income values in terms
+of how many standard deviations each value is from the mean.
```

Comment:
At this point, shouldn't we conclude something about the question we've been asking all along: whether KM or AOC's districts are particularly rich or poor?